src/org/mozilla/javascript/TokenStream.java

```diff
@@ -1194,11 +1259,33 @@ private static boolean isAlpha(int c)
         return 'a' <= c && c <= 'z';
     }
 
-    static boolean isDigit(int c)
+    private boolean isDigit(int base, int c) throws IOException {
```

Comment:
(It is already merged, but) I don't think `throws IOException` is necessary here.
Also, I think static method is better, just like `isDigit`.